Tasks/GooglePlayRelease/googleutil.ts

```diff
@@ -162,13 +162,15 @@ export async function getTrack(edits: any, packageName: string, track: string):
  * @param {string} track - one of the values {"internal", "alpha", "beta", "production"}
  * @param {integer or [integers]} versionCode - version code returned from an apk call. will take either a number or a [number]
  * @param {double} userFraction - for rollouting out a release to a track, it's the fraction of users to get update 1.0 is all users
+ * @param {priority} updatePriority - In-app update priority value of the release. All newly added APKs in the release will be considered at this priority. Can take values in the range [0, 5], with 5 the highest priority. Defaults to 0.
  * @param {releaseNotes} releaseNotes - optional release notes to be attached as part of the update
  * @returns {Promise} track - A promise that will return result from updating a track
  *                            { track: string, versionCodes: [integer], userFraction: double }
  */
-export async function updateTrack(edits: any, packageName: string, track: string, versionCode: any, userFraction: number, releaseNotes?: ReleaseNotes[]): Promise<Track> {
+export async function updateTrack(edits: any, packageName: string, track: string, versionCode: any, userFraction: number, updatePriority: number, releaseNotes?: ReleaseNotes[]): Promise<Track> {
```

Comment:
Unit test `updateTrack tests` (tasks/GooglePlayRelease/L0_googleutil.ts, line 94) fails soon after calling updateTrack. Could you please take a look?

![image](https://user-images.githubusercontent.com/72494759/113415201-65c33900-93c7-11eb-883e-feb1359dba74.png)

Reply:
fixed, I messed up parameters order 🙈